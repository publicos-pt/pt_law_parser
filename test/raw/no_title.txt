blabla
Artigo 1º
1 - As condições de trabalho constantes das alterações do contrato coletivo entre a APCOR - Associação Portuguesa da Cortiça e a FETESE - Federação dos Sindicatos da Indústria e Serviços, publicadas no Boletim do Trabalho e Emprego, nº 41, de 8 de novembro de 2014, são estendidas no território do continente:
a) Às relações de trabalho entre empregadores não filiados na associação de empregadores outorgante que se dediquem à atividade corticeira e trabalhadores ao seu serviço, das profissões e categorias profissionais nele previstas;
b) Às relações de trabalho entre empregadores filiados na associação de empregadores outorgante que prossigam a atividade mencionada na alínea anterior e trabalhadores ao seu serviço, das profissões e categorias profissionais nele previstas, não representados pela associação sindical outorgante.
