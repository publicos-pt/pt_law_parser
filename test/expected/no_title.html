<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
<p>blabla</p>
<div class="article" id="Artigo-1º"><h5 class="title">Artigo <a href="#Artigo-1º">1º</a></h5><ol><li class="number list-unstyled" id="Artigo-1º-Número-1">
<span><a href="#Artigo-1º-Número-1">1 -</a></span>
<span> As condições de trabalho constantes das alterações do contrato coletivo entre a APCOR - Associação Portuguesa da Cortiça e a FETESE - Federação dos Sindicatos da Indústria e Serviços, publicadas no Boletim do Trabalho e Emprego, nº 41, de 8 de novembro de 2014, são estendidas no território do continente:</span><ol><li class="line list-unstyled" id="Artigo-1º-Número-1-Alínea-a)">
<span><a href="#Artigo-1º-Número-1-Alínea-a)">a)</a></span>
<span> Às relações de trabalho entre empregadores não filiados na associação de empregadores outorgante que se dediquem à atividade corticeira e trabalhadores ao seu serviço, das profissões e categorias profissionais nele previstas;</span></li><li class="line list-unstyled" id="Artigo-1º-Número-1-Alínea-b)">
<span><a href="#Artigo-1º-Número-1-Alínea-b)">b)</a></span>
<span> Às relações de trabalho entre empregadores filiados na associação de empregadores outorgante que prossigam a atividade mencionada na alínea anterior e trabalhadores ao seu serviço, das profissões e categorias profissionais nele previstas, não representados pela associação sindical outorgante.</span></li></ol></li></ol></div></html>